src/features/dashboard/components/MemberDataPage.module.css

```
@@ -0,0 +1,152 @@
+	overflow: hidden;
+}
+.title {
+	margin-bottom: 1rem;
+}
+.panel {
+	display: flex;
+	flex-direction: column;
+	gap: 1.5rem;
+	padding: 2rem;
+
+	border-radius: 2rem;
```

Stylelint (17.13.0)

[error] Expected no empty line before declaration (declaration-empty-line-before)

(declaration-empty-line-before)